isisdaeApp/Db/isisdae.db

```diff
@@ -152,6 +160,7 @@ record(longin, "$(P)GOODFRAMES")
     field(SCAN, "I/O Intr")
     info(INTEREST, "HIGH")
 	field(EGU, "")
+    info(archive, "VAL")
```

Comment:
This will create a lot of logging details, we may need to use "scan" archiving - see comment on corresponding blockserver ticket